src/app/screen/input-housework/input-housework.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="home-container">
+  <div class="container">
+    <span>家事を入力してね</span>
+    <div class="underline"></div>
+    <div class="input-name"></div>
+    <div class="input-frequency"></div>
```

Comment:
同じような要素が続く場合は要素をまとめて扱ったほうが後々楽です。
以下のような構成にして、CSSも以下に合わせて修正してください。
```suggestion
    <div class="input-forms">
      <div class="input-name"></div>
      <div class="input-frequency"></div>
    </div>
```